erpnext_com/www/docs/user/manual/en/accounts/opening-balance.md

```diff
@@ -0,0 +1,69 @@
+<!-- add-breadcrumbs -->
+# Opening Balance in Accounts
+
+**The opening balance is the balance that is brought forward at the beginning of an accounting period from the end of a previous accounting period or when starting out.**
+
+## 1. Introduction
+
+If you are a new company you will have minimal opening balances to be imported. However, if you are migrating from a legacy accounting system like Tally or a Fox Pro based software you will have considerable data to be imported as opening balance.
+
+We recommend that you start using ERPNext for accounting from a new financial year, but you could start midway too. To setup your accounts, you will need the following for the “day” you start using accounting in ERPNext:
+
+##### Assets:
+1. Stock assets (stock in hand)
+1. Fixed assets like furnitures, computers etc
+1. Accounts Receivables(AR) i.e. unpaid invoices which you have sent to your customers
+1. Current assets like bank balances, cash in hand, deposits etc
+
+##### Liabilities:
+
+1. Capital accounts like your shareholder’s (or owner’s) capital
+1. Current liabilities like loans, salary payables etc
+1. Accounts Payables(AP) i.e. unpaid invoices which your suppliers have sent you
+
+
+If you were using another accounting software before, you should close financial statements in that software first. The closing balance of the accounts should be updated as an opening balance in the ERPNext. Before starting to update opening balance, ensure that your [Chart of Accounts](/docs/user/manual/en/accounts/chart-of-accounts) has all the Accounts required.
+
+> Opening entry is only for Balance Sheet accounts and not for Profit and Loss Accounts.
+
+## 2. Opening Balance of Assets
+
+2.1 [Fixed Assets](/docs/user/manual/en/accounts/opening-balance/fixed_assets)
+
+2.2 [Stock Assets](/docs/user/manual/en/stock/opening-stock)
+
+2.3 [Accounts Receivable](/docs/user/manual/en/accounts/opening-balance/accounts_receivable)
+
+2.4 [Current Assets](/docs/user/manual/en/accounts/opening-balance/current_assets)
+
+## 3. Opening Balance of Liabilities
+
+3.1 [Capital Accounts](/docs/user/manual/en/accounts/opening-balance/capital_accounts)
+
+3.2 [Current Liabilities](/docs/user/manual/en/accounts/opening-balance/current_liabilities)
+
+3.3 [Accounts Payable](/docs/user/manual/en/accounts/opening-balance/accounts_payable)
+
+## 4. Verify the Opening Balance
+
+Once all assets and liabilities have been imported, the balance of **Temporary Opening** ledger should be zero.
+
+After completing the accounting entries, the trial balance report will look as shown below.
+
+<img class="screenshot" alt="Opening Account" src="{{docs_base_url}}/assets/img/accounts/opening-4.png">
```

Comment:
image with this name not present in PR

Reply:
It already exists... not a new screenshot.

https://github.com/frappe/erpnext_com/blob/master/erpnext_com/www/docs/assets/img/accounts/opening-4.png